UNRELEASED.md

```diff
@@ -3,6 +3,13 @@
 > Place your changes below this line.
 >
 
+**Fixed:**
+
+- Updated Button component to match Design System:
+  - Updated various colours
+  - Updated icon spacing + horizontal padding
+  - Removed all caps property
```

Comment:
We need to mention that it's false by default now, as this is kinda breaking change.